packages/vm/lib/evm/evm.ts

```diff
@@ -218,14 +225,21 @@ export default class EVM {
     toAccount.nonce = new BN(toAccount.nonce).addn(1).toArrayLike(Buffer)
 
     // Add tx value to the `to` account
-    await this._addToBalance(toAccount, message)
+    let overflowError
+    try {
+      await this._addToBalance(toAccount, message)
+    } catch (e) {
+      overflowError = e
```

Comment:
A variable naming nit here:
`overflowError` variable could represent any other exception other than overflow, including unhandled ones (who knows?). As this variable just carries its value to the return statement, do you mind changing it to something more generic?